app/decorators/user_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class UserDecorator < Draper::Decorator
+  delegate_all
+
+def full_name
+	first_name = object.first_name = "first_name" ? "empty first name" : object.first_name
+	last_name = object.last_name = "last_name" ? "empty last_name" : object.last_name
```

Comment:
This is wrong. Homework - find difference between == and = 
Explain why it's wrong.